model/alert_test.go

```diff
@@ -116,3 +118,160 @@ func TestAlertValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestAlert(t *testing.T) {
+	// test unresolved alert
```

Comment:
Comments should be full sentences, starting with a capital letter and ending with a full stop.

Reply:
Sure!

Reply:
@brian-brazil Fixed